src/components/DiffView/index.tsx

```diff
@@ -0,0 +1,33 @@
+/// <reference path="index.d.ts"/>
+
+import * as React from 'react';
+import { Diff, ViewType, parseDiff } from 'react-diff-view';
+export interface Props {
```

Comment:
Why do we export the interface here?

Reply:
I exported everything by default, thinking we might need it in the tests. I will revisit what actually needs to be exported.